flink-connectors/flink-connector-cassandra/src/main/java/org/apache/flink/streaming/connectors/cassandra/CassandraSink.java

```diff
@@ -95,6 +95,28 @@ private CassandraSink(SingleOutputStreamOperator<IN> sink) {
 	}
 
 	/**
+	 * Sets an additional, user provided hash for this operator.
+	 *
+	 * <p/>
+	 * <p>The user provided hash is an alternative to the generated hashes, that is considered when identifying an
+	 * operator through the default hash mechanics fails (e.g. because of changes between Flink versions.
+	 *
+	 * <p><strong>Important</strong>: this hash needs to be unique per transformation and job. Otherwise, job
```

Comment:
If it is not true, then how can state be assigned without ambiguities if the hash is used as basis for JobVertexID?